atomicapp/nulecule/container.py

```diff
@@ -160,7 +160,7 @@ def extract(self, image, source, dest, update=False):
         Utils.rm_dir(tmpdir)
 
         # Clean up dummy container
-        rm_cmd = [self.docker_cli, 'rm', '-f', container_id]
+        rm_cmd = [self.docker_cli, 'rm', '-f', container_id[:12]]
```

Comment:
IMO using a CONSTANT rather than hard-coding the number would be a good thing to do.